webcompat/form.py

```diff
@@ -91,11 +100,11 @@ def get_form(ua_header):
     return bug_form
 
 
-def get_problem(category):
+def get_radio_button_label(field_value, label_list):
     '''Return human-readable label for problem choices form value.'''
-    for choice in problem_choices:
-        if choice[0] == category:
-            return choice[1]
+    for label in label_list:
+        if label[0] == field_value:
+            return label[1]
```

Comment:
(no reply needed) not sure if expected to provide a rationale but chose option 1 because it seemed more accessible for all levels of future contributors. think that options 2 and 3 are better for efficiency, however since the length of radio button options is relatively short, the first option should not be negative.